src/test/kotlin/racingcar/domain/result/CarSnapShotTest.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain.result
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import racingcar.domain.car.Car
+import racingcar.domain.car.vo.Name
+import racingcar.domain.car.vo.Position
+
+class CarSnapShotTest {
+    @Test
+    fun `스냅샷 생성`() {
+        val car = Car(Name("제이"), Position(10))
+        val carSnapShot = CarSnapShot(car)
+
+        assertThat(carSnapShot.name).isEqualTo("제이")
+        assertThat(carSnapShot.position).isEqualTo(10)
+    }
+
+    @Test
+    fun `동일한 위치인지 확인`() {
+        val car = Car(Name("제이"), Position(10))
+        val carSnapShot = CarSnapShot(car)
+
+        assertThat(carSnapShot.isSamePositionWith(10)).isTrue
+        assertThat(carSnapShot.isSamePositionWith(9)).isFalse
```

Comment:
```suggestion
        assertAll({
            assertThat(carSnapShot.isSamePositionWith(10)).isTrue
            assertThat(carSnapShot.isSamePositionWith(9)).isFalse
        })
```

여러개의 assertThat을 assertAll로 감싸줬을 때의 장점은 무엇일까요? 🤔

Reply:
assertAll을 쓰지 않으면, 예외가 발생했을 때 바로 테스트가 끝나버려서 다음 assert구문을 확인할 수가 없겠네요.
두 결과물이 연관되어 있지 않으면 assertAll을 쓰는 편이 좋아보이네요 :)